frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -87,7 +87,7 @@ const Metric = (props: MetricProps) => {
     }, [props.colorMap, props.max]);
 
     const data = useMemo(() => {
-        const mode = props.type === "none" ? [] : ["gauge"];
+        const mode = props.type == null || props.type.toLowerCase() === "none" ? [] : ["gauge"];
```

Comment:
what if props.type is undefined ?

Reply:
I'd like you to change the definition of the type attribute in the MetricsProps interface so that the test can pass